testsuite/openshift/mockserver.py

```diff
@@ -0,0 +1,38 @@
+"""Module for Mockserver integration"""
+import json
+from urllib.parse import urljoin
+
+import requests
+
+
+class Mockserver:
+    """Mockserver deployed in Openshift (located in Tools or self-managed)"""
+
+    def __init__(self, url):
+        self.url = url
+
+    def create_expectation(self, expectation_id, path, opa_policy):
+        """Creates an Expectation - response with body that contains OPA policy (Rego query)"""
+        response = requests.put(
+            urljoin(self.url, "/mockserver/expectation"), verify=False, timeout=5, data=json.dumps(
+                {
+                    "id": expectation_id,
+                    "httpRequest": {
+                        "path": path
+                    },
+                    "httpResponse": {
+                        "body": opa_policy
```

Comment:
The response must either be the raw Rego content of the policy (`Content-Type: text/plain`) or comply with OPA Policy API's [_Get a Policy_](https://www.openpolicyagent.org/docs/latest/rest-api/#get-a-policy) endpoint (`Content-Type: application/json`).

Reply:
I've added `"Content-Type": ["plain/text"]`
We will create tests for content-type in the future.